### ISSUE TITLE
[UX] - Confidence chart scale

### ISSUE THREAD

**Issue by jhoetter** [code-kern-ai/refinery#139]
**What would you improve?**
I'm not too sure about this one but I wanted to discuss it here. The confidence chart is currently maxed on the y-axis depending on the maximum confidence. If you have e.g. 80% as your maximum confidence, it will be capped at that value.

<img width="1305" alt="image" src="https://user-images.githubusercontent.com/62521561/195453182-d1102568-c9fe-4681-ad64-01a4b7004901.png">

It is actually showing a percentage, such that 100% could also be a valid choice. What I'm fearing if we don't do so, is that e.g. a maximum value at 30% confidence will look "good".

**Comment by JWittmeyer** [comment]
makes sense, so prevent irritation or misinterpretation we should fix the y axis to 100
